Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent none
+    stages {
+        stage("Analyse") {
+            agent {
+                label "sonar"
+            }
+            tools {
+                jdk "zulu-8"
+            }
+            steps {
+                echo "Analysing"
+                checkoutAndBuildSource()
+            }
+        }
+    }
+   post {
+        always {
+            hipchatSend (color: 'GRAY', credentialId: 'hipchat2', textFormat: false, message: "${env.JOB_NAME} build # ${env.BUILD_NUMBER} Completed. See <a href='${BUILD_URL}'>Jenkins</a>")
```

Comment:
I'm not sure what's in the job name, but will this message contain any JIRA specific reference? I.e. the PR name or something?

Reply:
Good point, that would be handy, but then I don't think the hipchat message is even necessary for PR analysis. It would get annoying quickly. Maybe in the future it could be set up to message the PR author.